blog/2024-07-09-boa-release-19.md

```diff
@@ -0,0 +1,144 @@
+<!-- ---
+layout: post
+tags: [post]
+title: "Boa release v0.19"
+author: Boa Developers
+--- -->
+
+## Summary
+
+Boa v0.19 is now available! After 4 months of development we are very happy to present you the latest
+release of the Boa JavaScript engine. Boa makes it easy to embed a JS engine in your projects, and
+you can even use it from WebAssembly. See the [about](/about) page for more info.
+
+In this release, our conformance has grown from 85.03% to 86.20% in the official ECMAScript Test Suite
+(Test262). Interestingly, this isn't due to us passing more tests, but in fact the total number of tests from test262
+[reducing by around 2000](https://github.com/tc39/test262/commit/ea2268aa4382013f5533b91f9ef50366ad065a86) from the [removal of custom calendars and timezones](https://github.com/tc39/proposal-temporal/issues/2826). Overall, Boa's conformance in real terms has improved by ~6%.
+
+You can check the full list of changes [here][changelog], and the full information on conformance
+[here][conformance].
+
+## Highlights
+
+### Temporal
+
+Boa is continuing to progress on the [Temporal](https://github.com/tc39/proposal-temporal). The Temporal API is a new
+set of built-in objects and functions that is designed to be a more modern replacement for the `Date`
+object, providing a more feature-rich and flexible API for working with dates and times.
+
+It is currently a [stage 3 proposal](https://tc39.es/proposal-temporal/docs/) and we are working
+alongside the TC39 champions to put together a solid Rust implementation. Since Temporal is such an
+extensive specification, we have done most of the work outside of Boa so that it can be used in other
+projects. This work can be found in the [temporal_rs](https://github.com/boa-dev/temporal/) repository.
+
+We hope to release a full blog post on Temporal in the future, but for now you can see the previous release notes for some examples on how to use it.
+You can also look at the [Temporal Cook Book](https://tc39.es/proposal-temporal/docs/cookbook.html) for some examples too!
+
+If you're interested in learning more or want to contribute to the native Rust implementation of
+Temporal, feel free to check out `temporal_rs`'s [issues](https://github.com/boa-dev/temporal/issues)!
+
+Boa's conformance on the Temporal test suite has grown from 23.99% to 27.86% in this release.
```

Comment:
```suggestion
Boa's conformance on the Temporal test suite has grown from 19.35% to 24.61% in this release.
```

Was off a bit on these / the newest updates from last night increased conformance, and noticed I forgot to add back in the passed tests to the total :sweat_smile: 

These are based off the `main` and v0.18.0 on our conformance page.